docs/vtex-io/Getting-Started/back-end-development-guides/service-path-patterns.md

```diff
@@ -6,24 +6,34 @@ hidden: false
 
 When [developing a new VTEX IO service](https://developers.vtex.com/docs/guides/vtex-io-documentation-developing-service-configuration-apps), you must define a path so that the service is available to receive requests.
 
+
 In this article, you will learn about the pattern for service paths according to the desired behavior. To know more about how to implement the path in your app's code, check the guide [Developing service configuration apps](https://developers.vtex.com/docs/guides/vtex-io-documentation-developing-service-configuration-apps).
 
+
 ## Patterns
 
+
 There are three possible path formats, depending on the behavior you wish for your service.
 
+
 For the purpose of this tutorial, consider you are implementing the service `/examplepath/123`. You may change how VTEX handles requests to this service by adding `/_v/segment` or `/_v/private` to the beginning of your path. This means you have three possible paths in this example:
 
+
 - **Public:** `/examplepath/123`
 - **Segment dependant:** `/_v/segment/examplepath/123`
 - **Private:** `/_v/private/examplepath/123`
 
+
 VTEX has an edge layer that handles all requests to the platform, including VTEX IO apps, in order to improve performance. Because of this, VTEX will handle requests to each of these possible paths differently when it comes to caching and handling cookies. See the table below.
 
 >ℹ️ Learn more about [VTEX cookies](https://developers.vtex.com/docs/guides/sessions-system-overview).
 
+>⚠️ Custom endpoints must follow the same path patterns to access cookies.
```

Comment:
Não entendi esse ponto 🤔

Reply:
Se for utilizada alguma rota diferente de `/_v/private`, `/_v/public` ou `/_v/segment`, ou seja, um endpoint customizado, não temos controle sobre o comportamento dos cookies. Isso porque esses paths já estão incluídos nas requests de GraphQL de acordo com o cache control declarado no GraphQL schema. Assim, não há garantia de que custom endpoints, como por exemplo `/_v/{custom-path}`,  irão ter acesso aos cookies desejados. Desta forma, o recomendado é seguir com os path patterns citados, de acordo com a definição de cada um e o que é desejado pela loja.